packages/frontend/web/server/htmlTemplate.ts

```diff
@@ -64,6 +78,14 @@ export const htmlTemplate = ({
 
                 ${fontPreloadTags.join('\n')}
 
+                ${openGraphMetaTags}
+
+                ${twitterMetaTags}
+
+                <meta name="twitter:image" content="${
+                    twitterData['twitter:image']
```

Comment:
If there is no twitterData (🤷‍♂ not sure if this can happen) then this resolves to `<meta name="twitter:image" content="undefined"/>` which is fine, but might be good to guard against it.